src/compiler/directives/model.js

```diff
@@ -69,6 +69,7 @@ type ModelParseResult = {
 
 export function parseModel (val: string): ModelParseResult {
   len = val.length
+  val = val.trim()
```

Comment:
Shouldn't this be above the previous line 🤔

Reply:
It's true, I fixed this in the last commit